temp_library/helpers.py

```diff
@@ -3,117 +3,103 @@
     input("press enter to roll...")
-    print("your " + trait3.getName() + " is in total: " + str(trait3.getValue()))
+    print("your " + trait3.get_name() + " is in total: " + str(trait3.get_value()))
 
     return trait1, trait2, trait3
 
 
 def statsdsply(trait1, trait2, trait3):
     # displaying the stats function
     print(' ------------------------------')
-    print('|'+trait1.getName()+': ' + str(trait1.getValue()) + '|'+trait2.getName()+': '+str(trait2.getValue())+'|'+trait3.getName()+': '+str(trait3.getValue())+'|')
+    print('|'+trait1.get_name()+': ' + str(trait1.get_value()) + '|'+trait2.get_name()+': '+str(trait2.get_value())+'|'+trait3.get_name()+': '+str(trait3.get_value())+'|')
     print(' ------------------------------')
     print()
 
 
-def roll(num, arg):
+def roll(trait, arg):
     # dice rolling function
     input("press enter to roll the dice...")
-    dice = 0
-    for i in range(1):
-        dice += random.randint(1, 6)
+    dice_roll = 0
+    dice_num = trait.get_dice_num()
+    trait_value = trait.get_value()
+
+    lucky_message = "success - press enter to continue..."
+    unlucky_message = "you were unlucky this time. press enter to continue..."
+
+    for i in range(dice_num):
+        dice_roll += random.randint(1, 6)
+
+    output = "you rolled a " + str(dice_roll)
 
     if arg == 'less':
-        if dice <= num:
-            print("")
-            print("you rolled a " + str(dice))
-            print("")
-            input('success - press enter to continue...')
-            return True
+        # lucky
+        if dice_roll <= trait_value:
+            return True, output, lucky_message
         else:
-            print("")
-            print("you rolled a " + str(dice))
-            print("")
-            input('you were unlucky this time. press enter to continue...')
-            return False
+            # unlucky
+            return False, output, unlucky_message
     else:
-        if dice >= num:
-            print("")
-            print("you rolled a " + str(dice))
-            print("")
-            input('success - press enter to continue...')
-            return True
+        if dice_roll >= trait_value:
+            # lucky
+            return True, output, lucky_message
         else:
-            print("")
-            print("you rolled a " + str(dice))
-            print("")
-            input('you were unlucky this time. press enter to continue...')
-            return False
+            # unlucky
+            return False, output, unlucky_message
 
 
 def fight(skill, stamina, m_skill, m_stamina):
     # fighting function
     print("your stamina "+str(stamina)+"monster stamina "+str(m_stamina))
-    Tskill = skill + random.randint(1, 6)
-    print("your skill is "+str(Tskill))
-    Tm_skill = m_skill + random.randint(1, 6)
-    print("monster skill is " + str(Tm_skill))
+    temp_skill = skill + random.randint(1, 6)

```

Comment:
Start thinking how you would implement this as classes (monster->defend? hero->attack? stuff like that - maybe you will have a better idea too :) )